src/packagedcode/spec.py

```diff
@@ -0,0 +1,153 @@
+# All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-toolkit/
+# The ScanCode software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode require an acknowledgment.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with ScanCode or any ScanCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import OrderedDict
+import csv
+import glob
+import io
+import logging
+import os
+import re
+
+from gemfileparser import GemfileParser
+
+"""
+Handle Cocoapods(.podspec) and Ruby(.gemspec) files.
+"""
+
+TRACE = False
+
+logger = logging.getLogger(__name__)
+
+if TRACE:
+    import sys
+    logging.basicConfig(stream=sys.stdout)
+    logger.setLevel(logging.DEBUG)
+
+
+def parse_spec(location):
+    """
+    Return dictionary contains podspec or gemspec file data.
+    """
+    with io.open(location, encoding='utf-8', closefd=True) as data:
+        lines = data.readlines()
+
+    spec_data = {}
+
+    for line in lines:
+        if '.name' in line:
+            name = line.rpartition('=')
+            spec_data['name'] = get_stripped_data(name[2])
+        elif '.version' in line and '.version.' not in line:
+            version = line.rpartition('=')
+            spec_data['version'] = get_stripped_data(version[2])
+        elif '.license' in line:
+            license_type = line.rpartition('=')
+            spec_data['license'] = get_stripped_data(license_type[2])
+        elif '.email' in line:
```

Comment:
We have authors and emails (plural):
https://github.com/search?q="authors"+extension%3A.gemspec&type=Code
https://github.com/search?q="emails"+extension%3A.gemspec&type=Code

Reply:
We are using a concept that a particular string is present in the line or not. and I think `.author` will work for both `authors` and `author` and similar to the email also. I think we will not have any error by using this. Can you tell me a case when it will detect wrong??